benchmarks/litdata/stream_raw_imagenet.py

```diff
@@ -0,0 +1,144 @@
+"""Benchmark streaming raw dataset with litdata."""
```

Comment:
Would you mind adding a section on the README for this dataset. 

Basically, when would you want to this vs StreamingDataset.